rpcserver.go

```diff
@@ -2224,6 +2224,28 @@ func (r *rpcServer) AddrReceives(ctx context.Context,
 
 	var sqlQuery address.EventQueryParams
 
+	if req.Offset < 0 {
+		return nil, fmt.Errorf("offset must be non-negative")
+	}
+	if req.Limit < 0 {
```

Comment:
Should there be a maximum limit enforced here as well? E.g. the universe code checks that the limit doesn't exceed a max page size (L6481):

```
	if req.Limit > universe.MaxPageSize || req.Limit < 0 {
		return nil, fmt.Errorf("invalid request limit: %d", req.Limit)
	}
```